docs/metadata/git.md

```diff
@@ -1,72 +1,67 @@
+<iframe src="https://user-images.Githubusercontent.com/121796483/216262458-a2c4a45b-d999-4fe8-99eb-f39a494062ca.mp4" title="External Git" allow="autoplay;fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
 </div></div>
 
 Every user will have a `Prophecy Managed` Git provider already created in their account. You can just use this if you don't want to store your code to an external Git.
 
-## How to create a new project with external Git provider
-
-Once a Git Credential is created, these credentials can be used during project creation.
+## How to create a new Project with external Git provider
 
-Please refer below video for step-by-step example
+Once a Git Credential is created, these credentials can be used during Project creation (video below). Later, these credentials will be used to commit and release changes to Git for the entire Project.
 
 <div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
 <div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
-<iframe src="https://user-images.githubusercontent.com/121796483/216262055-a2507bfa-b363-4e41-8a32-c11896606173.mp4" title="External Git" allow="autoplay;fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+<iframe src="https://user-images.Githubusercontent.com/121796483/216262055-a2507bfa-b363-4e41-8a32-c11896606173.mp4" title="External Git" allow="autoplay;fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
 </div></div>
 
-If you have not created your Git credentials already, you also have an option to create a new one from the project creation screen directly. They will also start reflecting in your Git screen and can be reused further for new projects.
+If you have not created your Git credentials already, you also have an option to create a Git credential from the Project creation screen directly. This Git credential can be reused for new Projects.
 
 ## How to commit changes
 
-When you create your first project, `dev` branch is automatically created for you. This is to ensure that no edits/commits are
-made directly on the `main` branch and a proper CICD flow is followed during development.
+When you create your first Project, `dev` branch is automatically created for you. This is to ensure that no edits/commits are made directly on the `main` [branch](http://localhost:3000/concepts/Project/#Project-commits).
 
-Creation of first Pipeline and commit changes to `dev` branch:
+Within a Project, create a Pipeline and commit changes to a feature branch. The default feature branch is called `dev`:
```

Comment:
It's not a feature branch. It's a user branch. It's not named `dev` by default, it's `dev/$USERNAME`